manager/integration/tests/test_basic.py

```diff
@@ -886,15 +888,15 @@ def test_listing_backup_volume(clients, base_image=""):   # NOQA
     for i in range(RETRY_COUNTS):
         bvs = client.list_backupVolume()
         for bv in bvs:
-            if bv["name"] == volume1_name:
-                if "error" in bv.messages:
-                    assert "volume.cfg" in bv.messages["error"].lower()
+            if bv.name == volume1_name:
+                if "error" in bv.messages.__dict__:
```

Comment:
Why we need `__dict__` here?

Reply:
`bv.messages` is of type `longhorn.RestObject` and is not iteratable.

>*** TypeError: argument of type 'RestObject' is not iterable

Reply:
Can you try the following?
```
diff --git a/manager/integration/tests/longhorn.py b/manager/integration/tests/longhorn.py
index 1620567..faeaa35 100644
--- a/manager/integration/tests/longhorn.py
+++ b/manager/integration/tests/longhorn.py
@@ -134,6 +134,7 @@ class RestObject:
     def __iter__(self):
         if self._is_list():
             return iter(self.data)
+        return iter(self.__dict__)
```

Reply:
Per https://docs.python.org/3/reference/expressions.html#membership-test-operations

```
For user-defined classes which define the __contains__() method, x in y returns True if y.__contains__(x) returns a true value, and False otherwise.

For user-defined classes which do not define __contains__() but do define __iter__(), x in y is True if some value z, for which the expression x is z or x == z is true, is produced while iterating over y. If an exception is raised during the iteration, it is as if in raised that exception.

Lastly, the old-style iteration protocol is tried: if a class defines __getitem__(), x in y is True if and only if there is a non-negative integer index i such that x is y[i] or x == y[i], and no lower integer index raises the IndexError exception. (If any other exception is raised, it is as if in raised that exception).
```

We do defined `__getitem__()` and `__iter__()` in the object, why `in` doesn't work?

Reply:
Adding `return iter(self.__dict__)` to `__iter__` definition works.